pkgs/tools/text/agrep/default.nix

```
@@ -1,33 +1,42 @@
-{ lib, stdenv, fetchFromGitHub }:
-
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+let
+  # This repository has numbered versions, but not Git tags.
+  rev = "b7d180fe73636740f694ec60c1ffab52b06e7150";
+in
 stdenv.mkDerivation {
   pname = "agrep";
   version = "3.41.5";
```

```suggestion
  version = "3.41.5-unstable-2022-03-23";
```
Please add `-unstable-YYYY-MM-DD` when we use HEAD.
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming